documentation/azure-powershell-modules.md

```diff
@@ -0,0 +1,110 @@
+# Azure PowerShell
+
+## Rollup Modules
+
+| Description                                    | Module Name       | PowerShell Gallery Link |
+| ---------------------------------------------- | ----------------- | ----------------------- |
+| Rollup Module for ARM Cmdlets                  | `AzureRM`         | [![AzureRM](https://img.shields.io/powershellgallery/v/AzureRM.svg?style=flat-square&label=AzureRM)](https://www.powershellgallery.com/packages/AzureRM/) |
+| Rollup Module for .NET Core Cmdlets            | `AzureRM.Netcore` | [![AzureRM.Netcore](https://img.shields.io/powershellgallery/v/AzureRM.Netcore.svg?style=flat-square&label=AzureRM.Netcore)](https://www.powershellgallery.com/packages/AzureRM.Netcore/) |
+| Rollup Module for Azure Stack Cmdlets          | `AzureStack`      | [![AzureStack](https://img.shields.io/powershellgallery/v/AzureStack.svg?style=flat-square&label=AzureStack)](https://www.powershellgallery.com/packages/AzureStack/) |
```

Comment:
Same.  AzureStack is the rollup for administrative (rather than control plane) modules in AzureStack